src/backend/chat_service/chat_service.py

```diff
@@ -141,13 +141,16 @@ async def list_files(request: Request, customer_guid: str):
             logger.info(f"No files found in bucket '{customer_guid}'")
 
         return {"files": file_list}
-    except Exception as e:
-        logger.error(f"Error listing files:'{customer_guid}': {e}")
-        raise HTTPException(status_code=500, detail="Error listing files")
+    except HTTPException as e:
+        if e.status_code==404:
+            logger.error(f"Bucket not found for customer_guid '{customer_guid}': {e.detail}")
```

Comment:
Change this to a common log instead of being specific "Bucket not found for customer_guid"